agent/containers/images/Makefile

```diff
@@ -108,7 +83,7 @@ _ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
 _DEFAULT_DISTROS = centos-7 centos-8 centos-9 fedora-35 fedora-36
 
 # By default we build the Tool Data Sink and Tool Meister images as well.
-all: all-tags $(_DEFAULT_DISTROS:%=%-all-tagged)
+everything: all-tags $(_DEFAULT_DISTROS:%=%-all-tagged)
```

Comment:
How do this change improve anything?  (AFAIK, `make all` is a more common idiom than `make everything`...here's [one set of conventions](https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html#Standard-Targets)....)

This change doesn't address the problem which [I outlined above](https://github.com/distributed-system-analysis/pbench/pull/3018#issuecomment-1258434255), which is with the `%-all` target, not the default, `all` target.

Reply:
As I wrote in the commit message, we have a container kind or type named `all`.  It seemed to me that that caused all this confusion.  So by renaming the `all` target to `everything` we remove that confusion.

Reply:
The problem seems to stem from the conflict between the `make` conventions regarding the token `all` and the use of that token to name one of the containers in the distro.  Since it will be disruptive to rename that container, we're left either with bucking the `make` conventions or having an inconsistency between the mid-level targets and the artifacts that they build.

Changing the top-level target is undesirable, but it seems to be the least of evils.  The default target doesn't need to be explicitly specified, so having it named `everything` instead of `all` is not a huge problem (since people can just omit it), particularly now that the Makefile will object to the use of the `all` target.